mne/channels/montage.py

```diff
@@ -505,7 +506,12 @@ def make_dig_montage(ch_pos=None, nasion=None, lpa=None, rpa=None,
     read_dig_montage
 
     """
-    ch_names = None if ch_pos is None else list(sorted(ch_pos.keys()))
+    if ch_pos is None:
+        ch_names = None
+    else:
+        ch_names = list(ch_pos.keys())
+        if not isinstance(ch_pos, OrderedDict):
+            ch_names = sorted(ch_names)
```

Comment:
If we make them all OrderedDicts, when could they not be OrderedDict? what am I missing?

Reply:
> when could they not be OrderedDict? what am I missing?

If a user uses the `make_dig_montage` function themselves